src/Map/src/Bridge/Google/assets/dist/map_controller.js

```diff
@@ -1,8 +1,48 @@
-import AbstractMapController from '@symfony/ux-map/abstract-map-controller';
+import { Controller } from '@hotwired/stimulus';
```

Comment:
Surely because Rollup has been able to resolve the file when using `import AbstractMapController from '@symfony/ux-map'`, the `AbstractMapController` is now inlined.

I'm fine with this, especially it removes one useless HTTP request.